src/Backend/Core/Layout/Css/screen.css

```diff
@@ -8074,8 +8074,8 @@ div.tabs ul li.selected {
 
 .bootstrap-tagsinput {
   display: block;
-  padding: 4px 6px 7px;
-  overflow: hidden; }
+  overflow: hidden;
```

Comment:
Following the scss-lint standard makes it easier for multiple people to work on the same code.